distribute.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -euo pipefail
+IFS=$'\n\t'
+mkdir dist
+echo "Building Way Cooler..."
```

Comment:
I was thinking it would be convenient to set the build system up to be triggerable for testing purposes, i.e. it would create GitHub releases just to confirm things work. We could either have it mark them (I was thinking we could call it "prereleases" but that term is overloaded) or just delete them after they're created. This would be a lot easier if the github releases part of the code wasn't just magically done by Travis.